src/pages/Admin/Charity/Donations/Table.tsx

```diff
@@ -29,133 +28,130 @@ export default function Table({
         >
-          {sorted
-            .map(({ hash, amount, symbol, chainId, date, kycData }) => (
-              <Cells key={hash} type="td" cellClass="p-2 first:pl-0 last:pr-0">
-                <>{humanize(amount, 3)}</>
-                <span className="text-sm">{symbol}</span>
+          <HeaderButton
+            onClick={handleHeaderClick("date")}
+            _activeSortKey={sortKey}
+            _sortKey="date"
+            _sortDirection={sortDirection}
+          >
+            Date
+          </HeaderButton>
+          <HeaderButton
+            onClick={handleHeaderClick("symbol")}
+            _activeSortKey={sortKey}
+            _sortKey="symbol"
+            _sortDirection={sortDirection}
+          >
+            Currency
+          </HeaderButton>
+          <HeaderButton
+            onClick={handleHeaderClick("amount")}
+            _activeSortKey={sortKey}
+            _sortKey="amount"
+            _sortDirection={sortDirection}
+          >
+            Total Amount
+          </HeaderButton>
+          <>Current Portion</>
+          <>Sustainability Portion</>
+          <>Transaction</>
+          <>Receipt provided</>
+        </Cells>
+      </TableSection>
+      <TableSection
+        type="tbody"
+        rowClass="even:bg-orange-l6 dark:odd:bg-blue-d6 dark:even:bg-blue-d7 divide-x divide-prim"
+        selectedClass="bg-orange-l5 dark:bg-blue-d4"
+      >
+        {sorted
+          .map(
+            ({
+              hash,
+              amount,
+              symbol,
+              chainId,
+              date,
+              kycData,
+              splitLiq = "50",
+            }) => (
+              <Cells
+                key={hash}
+                type="td"
+                cellClass={`p-3 border-t border-prim max-w-[256px] truncate ${
+                  hasMore ? "" : "first:rounded-bl last:rounded-br"
+                }`}
+              >
                 <span className="text-sm">
                   {new Date(date).toLocaleDateString()}
                 </span>
-                <ExtLink
-                  //default to ethereum for staging
-                  href={getTxUrl(chainId === "staging" ? "1" : chainId, hash)}
-                  className="text-center text-blue hover:text-blue-l2 cursor-pointer uppercase text-sm"
-                >
-                  {maskAddress(hash)}
-                </ExtLink>
-                {!kycData ? (
-                  <Icon type="CloseCircle" className="text-2xl text-red-400" />
-                ) : (
-                  <Icon
-                    type="CheckCircle"
-                    className="text-2xl text-green-400"
-                  />
-                )}
-                <></>
+                <span className="text-sm">{symbol}</span>
+                <>{humanize(amount, 3)}</>
+                <>{humanize(amount * (+splitLiq / 100), 3)}</>
+                <>{humanize(amount * (+splitLiq / 100), 3)}</>
```

Comment:
The calculations for `Current Portion` and `Sustainability Portion` are the same, is this intentional?

Reply:
nope, should be complement - fixing now. Also attached linear ticket